### PR TITLE
Adding first model

### PR DESCRIPTION
**Pull request by macumber**

### CODE REVIEW

File: models/ShoeboxExample/reports/eplustbl.html
Comment (jmarrec) [inline models/ShoeboxExample/reports/eplustbl.html:1]: Did you mean to commit the reports?!

Comment (macumber) [reply models/ShoeboxExample/reports/eplustbl.html:1]: Nope, thanks!

Comment (macumber) [reply models/ShoeboxExample/reports/eplustbl.html:1]: Might want to squash merge this and try to purge the git index of those big files

**Comment by macumber** [comment]
Force pushed a change removing large files, new PR is at https://github.com/openstudiocoalition/openstudio-coalition-measures/pull/2

File: models/ShoeboxExample/reports/eplustbl.html
Comment (macumber) [reply models/ShoeboxExample/reports/eplustbl.html:1]: Hmm, I did a force push to remove the files which invalidated this branch, new pr is at #2

**Pull request closed without merging**